routes/books.js

```diff
@@ -29,6 +43,12 @@ router.get('/:id', (req, res, next) => {
   })
 })
 
```

Comment:
See if you can refactor `checkInput` so it's similar to [how I used `validateId` over in the solution branch](https://github.com/gSchool/g59-dry-refactor/blob/f6687478d3f2f4b039674fc9e76961f4199ffd3d/routes/books.js#L29-L33)

Note all routes with `/:id` use the same validate technique.